spec/dummy/db/schema.rb

```diff
@@ -13,12 +13,12 @@
 
 ActiveRecord::Schema.define(version: 20150115103932) do
 
-  create_table "blog_images", force: true do |t|
+  create_table "blog_images", force: :cascade do |t|
```

Comment:
This is why: http://edgeguides.rubyonrails.org/4_2_release_notes.html#active-record-notable-changes

```
SchemaDumper uses force: :cascade on create_table. This makes it possible to reload a schema when foreign keys are in place.
```